oauth_dropins/appengine_config.py

```diff
@@ -90,6 +91,7 @@ def read(filename):
 FLICKR_APP_SECRET = read('flickr_app_secret')
 GOOGLE_CLIENT_ID = read('google_client_id')
 GOOGLE_CLIENT_SECRET = read('google_client_secret')
+INDIEAUTH_CLIENT_ID = read('indieauth_client_id')
```

Comment:
I'm confused -- we give the wsgi application a class rather than an instance:

```python
    ('/indieauth/start', indieauth.StartHandler.to('/indieauth/oauth_callback')),
    ('/indieauth/oauth_callback', indieauth.CallbackHandler.to('/')),
```

I don't see where to pass parameters, unless I make another wrapper class definition like ToHandler, which seems like overkill.